UnityProject/Assets/Scripts/Detective/AppliedDetail.cs

```diff
@@ -0,0 +1,92 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AppliedDetail
```

Comment:
As this contains a collection of details, how about calling it `AppliedDetails`?